spec/classes/plugin/dns_cloudflare_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'letsencrypt::plugin::dns_cloudflare' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os} based operating systems" do
+      let(:facts) { os_facts }
+      let(:params) { { 'api_token' => 'dummy-cloudflare-api-token' } }
+      let(:pre_condition) do
+        <<-PUPPET
+        class { 'letsencrypt':
+          email => 'foo@example.com',
+        }
+        PUPPET
+      end
+      let(:package_name) do
+        osname = facts[:os]['name']
+        osrelease = facts[:os]['release']['major']
+        osfull = "#{osname}-#{osrelease}"
+        if %w[RedHat-7 CentOS-7].include?(osfull)
+          'python2-certbot-dns-cloudflare'
+        elsif %w[Fedora].include?(osname) || %w[Debian RedHat].include?(facts[:os]['family'])
```

Comment:
Fedora is in the RedHat OS family
```suggestion
        elsif %w[Debian RedHat].include?(facts[:os]['family'])
```

Reply:
Removed